scientific/src/commonMain/kotlin/formatter/CommonScientificValueFormatter.kt

```diff
@@ -40,49 +57,81 @@ sealed class CommonScientificValueFormatter private constructor(builder: Builder
              * @param build method for configuring the [Builder]
              * @return the built [ScientificValueFormatter]
              */
+            @Deprecated("Use CommonScientificValueFormatter.where", replaceWith = ReplaceWith("CommonScientificValueFormatter.where { build }"))
             fun build(build: Builder.() -> Unit = {}): ScientificValueFormatter {
                 val builder = Builder()
-                build(builder)
-                return Buildable(builder)
+                return builder.build()
             }
         }
 
-        internal val customFormatters = mutableMapOf<ScientificUnit<*>, CustomFormatHandler>()
+        var defaultValueFormatter = CommonScientificValueFormatter.defaultUnitFormatter
```

Comment:
Should `defaultValueFormatter` be public var?

Reply:
Yes otherwise it cannot be customized. This is the builder, so it can only be set within the scope of `CommonScientificValueFormatter.with {}`